en/services/parachute.md

```diff
@@ -0,0 +1,158 @@
+| 6  | 64     | `PARACHUTE_TRIGGER_FLAGS_ATS_AUTO_ARM`        | Auto-arming of parachute automatic trigger system (ATS). This allows a parachute to enable ATS after reaching a desired altitude. |
+| 7  | 128    | `PARACHUTE_TRIGGER_FLAGS_ATS_AUTO_DISARM`     | Auto-disarming of parachute automatic trigger system (ATS). This allows a parachute to disable ATS after detecting that it is below a desired altitude. |
+
+#### PARACHUTE_DEPLOYMENT_TRIGGER
+[PARACHUTE_DEPLOYMENT_TRIGGER](#PARACHUTE_DEPLOYMENT_TRIGGER) is an enum that details out the deployment status of the parachute. This enum is set to "0" when the parachute has not been deployed. All other enum values indicate that the parachute has deployed and by which source. This is published in the [PARACHUTE_STATUS](#PARACHUTE_STATUS) message.
+
+A list of the various enum values can be found below:
+
+| Value | Parameter                                          | Description                                                             |
+|-------|----------------------------------------------------|-------------------------------------------------------------------------|
+| 0     | `PARACHUTE_DEPLOYMENT_TRIGGER_NONE`                | None (parachute has not deployed)                                       |
+| 1     | `PARACHUTE_DEPLOYMENT_TRIGGER_MANUAL`              | Manual                                                                  |
+| 2     | `PARACHUTE_DEPLOYMENT_TRIGGER_ATS`                 | Automatic trigger system (ATS)                                          |
+| 3     | `PARACHUTE_DEPLOYMENT_TRIGGER_DRONE`               | Drone                                                                   |
+| 4     | `PARACHUTE_DEPLOYMENT_TRIGGER_MAVLINK`             | MAVLink, e.g. from an offboard computer                                 |
+| 5     | `PARACHUTE_DEPLOYMENT_TRIGGER_GEOFENCE`            | Geofence                                                                |
+| 6     | `PARACHUTE_DEPLOYMENT_TRIGGER_FTS_PRECHECKING`     | Flight Termination System (FTS) pre-checking protocol                   |
+
+
+#### PARACHUTE_SAFETY_FLAGS
+[PARACHUTE_SAFETY_FLAGS](#PARACHUTE_SAFETY_FLAGS) is a bitmask that details the safety status of the parachute. This is published in the [PARACHUTE_STATUS](#PARACHUTE_STATUS) message.
+
+A breakdown of the bitmask can be found below:
+
+| Value | Parameter                                             | Description                                                                                                     |
+|-------|-------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
+| 1     | `PARACHUTE_SAFETY_FLAGS_GROUND_CLEARED`               | The parachute module has cleared a safe distance from the ground for deployment.  |
```

Comment:
if I was a user reading this documentation, my first question would be "what does safe distance mean? Does this mean that I can't trigger the parachute through any means if this is not set, even if it's armed?"

Reply:
added additional info but it is custom set per parachute